.github/workflows/main.yml

```diff
@@ -0,0 +1,36 @@
+name: Tests
+on:
+  # Test all pull requests, and any push to master.
+  pull_request:
+  push:
+    branches:
+      - master
+  schedule:
+    # run once a month on master branch
+    - cron: '0 12 10 * *'
```

Comment:
I always see this same timestamp in every workflow... :)

Reply:
I saw '0 12 * * 0'  for running once a week on Sunday. I do it once a month, on day 10. So it is different. :-)
But yes, might be better to randomise this a bit. I will do that.